helper/date_format_test.go

```diff
@@ -18,6 +18,19 @@ func TestDateFormat(t *testing.T) {
 	})
 }
 
+func TestTimeFormat(t *testing.T) {
+	Convey("Time format returns time when passed date-time", t, func() {
```

Comment:
I would split this in 2 separate conveys: 24h and 12h

Reply:
i agree with your suggestion of splitting the functions into 2 separate conveys. Now we have functions called TimeFormatHH and TimeFormathh. The lower case hh is in 12h while the HH is in 24h.